quantecon/version.py

```diff
@@ -1,4 +1,4 @@
 """
 This is a VERSION file and should NOT be manually altered
 """
-version = '0.5.1'
+version = '0.5.2'
```

Comment:
This should not be manually altered? @mmcky

Reply:
This file is generated by the `setup.py` function 

https://github.com/QuantEcon/QuantEcon.py/blob/680ab7456f006153fe7d67337e2e42a304edd063/setup.py#L9

Probably better ways to do this but when I setup this package initially this is way we configured it.

Reply:
so the version should be updated https://github.com/QuantEcon/QuantEcon.py/blob/680ab7456f006153fe7d67337e2e42a304edd063/setup.py#L7

Reply:
So it should have been altered automatically, but it wasn't?

Reply:
Hmm -- 🤔 -- it should update during a release. Thanks @oyamad. @Smit-create this can remain as it is. The warning is only there as it "should" get overridden.

Reply:
So, should I revert the change back to '0.5.1'? or else it's fine? Though, I haven't updated it manually. I believe this was updated during `python setup.py install`

Reply:
Well, one suggestion to avoid editing this file manually can be:
```diff
diff --git a/quantecon/version.py b/quantecon/version.py
index eff1340..bf2fe97 100644
--- a/quantecon/version.py
+++ b/quantecon/version.py
@@ -1,4 +1,3 @@
-"""
-This is a VERSION file and should NOT be manually altered
-"""
-version = '0.5.2'
+import pkg_resources
+
+version = pkg_resources.require("quantecon")[0].version
```
I guess this will automatically find the version once the project is setup (we can remove `write_version_py` safely)

Reply:
@Smit-create great suggestion. Love it. Please go ahead.

Reply:
Thanks! Done